policies/redis/producer/streams.go

```diff
@@ -229,17 +229,17 @@ func (e eventStore) ValidatePolicy(ctx context.Context, token string, p policies
 	return e.svc.ValidatePolicy(ctx, token, p, format, policyData)
 }
 
-func (e eventStore) DeleteSinkFromDataset(ctx context.Context, sinkID string, token string) ([]policies.Dataset, error) {
-	return e.svc.DeleteSinkFromDataset(ctx, sinkID, token)
+func (e eventStore) DeleteSinkFromAllDatasetsInternal(ctx context.Context, sinkID string, token string) ([]policies.Dataset, error) {
+	return e.svc.DeleteSinkFromAllDatasetsInternal(ctx, sinkID, token)
 }
 
-func (e eventStore) InactivateDatasetByID(ctx context.Context, datasetID string, token string) error {
-	ds, err := e.svc.ViewDatasetByID(ctx, token, datasetID)
+func (e eventStore) InactivateDatasetByIDInternal(ctx context.Context, datasetID string, ownerID string) error {
+	ds, err := e.svc.ViewDatasetByIDInternal(ctx, ownerID, datasetID)
 	if err != nil {
 		return err
 	}
 
-	if err := e.svc.InactivateDatasetByID(ctx, datasetID, token); err != nil {
+	if err := e.svc.InactivateDatasetByIDInternal(ctx, datasetID, ownerID); err != nil {
```

Comment:
we see now that ViewDatasetByIDInternal takes (ctx, ownerID, datasetID) but InactivateDatasetByIDInternal takes (ctx, datasetID, ownerID)... we should instead keep the order of the arguments consistent, which makes for a nicer API to develop with helps avoid programmer error (because they are both strings, they are easily swapped). since ViewDatasetByIDInternal already exists, let's change InactivateDatasetByIDInternal to match

Reply:
oh I see, ok, changing it